llpc/test/shaderdb/PipelineGsTess_TestInOutPacking.pipe

```diff
@@ -81,18 +80,28 @@ entryPoint = main
 [GsGlsl]
 #version 450 core
 layout(triangles) in;
-layout(triangle_strip, max_vertices = 16) out;
-layout(location = 0) in vec4 in_vtxColor[];
-layout(location = 0) out vec4 vtxColor;
-out gl_PerVertex { vec4 gl_Position; float gl_PointSize; };
-in gl_PerVertex { vec4 gl_Position; float gl_PointSize; } gl_in[];
+layout(triangle_strip, max_vertices = 3) out;
+layout(location = 0) in vec2 in1[];
+layout(location = 1) in vec2 in2[];
+
+layout(stream = 0) out;
+layout(location = 0, xfb_buffer=1, xfb_offset=0) out ivec2 out1;
```

Comment:
what will happen if xfb offset isn't continous?
for example:
	layout(location = 0, xfb_buffer=1, xfb_offset=0) out int unused_output0;
	layout(location = 0, xfb_buffer=1, xfb_offset=8) out int unused_output1;
will you generate single tbuffer store with mask .x_z_ or two tbuffer_store instructions?

Reply:
we needn't pack stream out instructions. if you doesn't pack stream out instruction, I suppose you should not check stream out instruction in shaderdb test.

Reply:
Got it. I removed the check for stream out instructions.